discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
+
+* Clang Compiler & Tools
```

Comment:
This is pretty verbose, how about `Clang Frontend`?

Reply:
If we are naming this Clang Frontend, why are we putting other tools here then too? Should they just go under Subprojects then?